logic/src/jvmMain/kotlin/com/wire/kalium/logic/feature/call/GlobalCallManager.kt

```diff
@@ -3,10 +3,12 @@ package com.wire.kalium.logic.feature.call
 import com.wire.kalium.logic.data.call.CallRepository
 import com.wire.kalium.logic.data.client.ClientRepository
 import com.wire.kalium.logic.data.user.UserRepository
+import com.wire.kalium.network.api.NonQualifiedUserId
 
 actual class GlobalCallManager {
 
     actual fun getCallManagerForClient(
+        userId: NonQualifiedUserId,
```

Comment:
You can make this a `QualifiedID` not that https://github.com/wireapp/kalium/pull/335 is merged.

Reply:
Don't have to be in this PR.

Reply:
If you don't mind I will be doing this on the EPIC branch as I will need to probably solve the conflicts and thus already changing this to UserId

Reply:
changed it in this commit: https://github.com/wireapp/kalium/commit/833d363cef28612e61abf6184248aae6dbc24031